client/src/services/event-types/eventTypesService.js

```diff
@@ -1,21 +1,26 @@
 import requestService from '@/services/requestService';
 
+const apiEndpoint = '/event-types';
+
 const eventTypesService = {
-    async fetchAllEventTypes() {
-        const response = await requestService.get('/event-types');
+    async fetchAllEventTypes(searchString) {
+        const eventTypesApiUrl = searchString
+            ? apiEndpoint + '?searchString=' + searchString
+            : apiEndpoint;
+        const response = await requestService.get(eventTypesApiUrl);
```

Comment:
You need to pass parameters to `requestService` instead of adding them to the endpoint URL.